config/secrets.yml.enc

```diff
@@ -0,0 +1 @@
+rG/UdpwTxXIRZjJ4THuIquuPoUsZ73VKFgq/8pIzq8bTzBQe0Z44haJrmLr1xKoMHeSFCu8x0UHJA/Wfkcg1Qs0AxYr0d2BtEKet68OKlDozevo4OEvanoil4cVDh5p8d9JSQ0RfMwAv8N3WCebHyD6PeK1LCOSAkRwG/nH+ohPlm/tM4shzgNWWmgLjfe7ZzW14ItgbQ6+2xZXPCDIB+omasadMZWC963+3VW9diLvqWncpjqH2GDcgG3+ioCxxBX10IPlkA6jwo0fWRNfe8vCk5m+M2qdo7PUDiD8w76buymdNwBZ63Cls1s6JoK3zVRVzjwB7iHZRuEqZais4kxyGxPt1MSKzqPB1Fhnw0FcSrQR3hy1G7oj/eJckyJ7ehc+9zClU64UVKNq3IFDhRzi6MNx97jyKeDQeE3IMcOzg4U6Kc+Pic4yTXK/3oLTRNK1BtSCKlQIT70Q4R/OD3F3ylQ8f7iQtsNrkwMGUwTivXGOICExohYp6qfuL/V11Pp/z4A9isv0Hsl9GD4XtIuCiK54/dk3TeJei326CxTRm42k6+uUW5xXd7VH93wy15WViDEHlh9KUK3at19Mlzsm6+S+kRRU3e8jktge9nKzyrntD3hk8nZ8KWs++6m8nDz5z/4rm2+bC3adK5SsFH5rturNyVDn2TOZudjUyhcwvs9pgFrIIq+fk1YmgW+9znS+mHSYQ5OgAtfEIKplSSFnyNTH/1TJKMDJKUPpuBaXyKzfx9rWf1LjWN+o2AVm8ibOd0y7+u9BWY5GpFyWtVBiEf2mMS4LulrjPSWLf4GcK38oR7C9X/KblMHT5GGBmYy6FlupCXahLhpn/3TUoh48y1yXHoSMNoEyheefy9HonoOXJODgittEY+bdu7r6xmWP6szfHNO5C6X+uKGe80W6wVKTk3vsJWRy+1pdK4ix0aCWMMk16KqH97ZLCdg8KIg1TsZBhr8DbOYoTFVwLcOJzMs2UIQMNsotoO0yyZn9ng24sG7Sjtyl3IhEqibPTwz1OQdOqBqxZX9H/Z+lNol5LKPs81xDwvlzkAOo4jwRYtJJTkkTvLtobO7GvizfhuB4uLWXM73ycbpdW/0E6VyaAqeZOdF1mxw/D7nHTyZLPVqq4xCn6G+dJ0J+OQlcd7pz2K2tb0IyYfk1vs38=--xl5PAvsooSAzIFxY--zYG79IV6LAWKmaF/Ngb8HQ==
```

Comment:
Do we customarily encrypt our secrets rather than keeping them as environment variables?

Reply:
In the past we have not but Rails encrypted secrets is the new favored convention/system going forward and I quite like it. Instead of having to manually sling around config files with the secrets in them you just need to drop the key into the relevant server and you now have all the secrets enabled. Furthermore, instead of having to go in and edit changed secrets on the server every time you make a change, Capistrano and other deploy processes will automatically pull new/updated secrets from the repo. A new favorite feature of mine from Rails 5.1!